rustfmt.toml

```diff
@@ -0,0 +1,10 @@
+max_width = 100
+reorder_imports = true
+reorder_modules = true
```

Comment:
Is there a reason to specify these? They're all the default values according to https://rust-lang.github.io/rustfmt/?version=v1.6.0

Reply:
Oh, nice. These are coming from the template, so we should fix it there as well. I will remove them once I enable `nightly` fmt.